languages/python/exercises/concept/conditionals/.docs/introduction.md

```diff
@@ -0,0 +1,44 @@
+## Conditionals
+
+Conditionals are the most common statements used in programming. Conditionals determine the flow of control in a program. In python the `if` statement is used to control the flow of execution.
```

Comment:
@sachsom95 I would just add some words on the fact that the condition of an `if` statement must be of type `bool`, what do you think?

Reply:
yes Indeed I guess I took that for granted. Let me try to rephrase and add it to the content.